recipes/genepop/4.7.0/meta.yaml

```diff
@@ -0,0 +1,26 @@
+package:
+  name: genepop
+  version: "4.7.0"
```

Comment:
The version seems to be wrong, according to the link. Can you please use {{ verssion }} instead.

Reply:
Or the link is wrong :)

Reply:
@bgruening : First my apologies for taking so long.
In this case its neither: after talking with the author, we decided that for existing genepop versions we would take the out from the exinsting R packages. So, the R package is 1.0.5 but Genepop is 4.7.0. People do not really know the R package version.
In the future (hopefully) a standalone version of genepop will be provided (actually it is provided and always was, but the author did not kept old versions, just replaced the source code zip).
All this kerfufle is just to make sure we are able to maintain exact versions (reproducibility and all).
What probably makes sense is add this comment to the yaml, what do you think?

Reply:
Oh yes, please add this, otherwise it is very confusing :)

Reply:
Added the comment to the yaml file.